apps/sms/views/shared/js/localization_helper.js

```diff
@@ -11,11 +11,13 @@
       'iframe.attachment-container'
     );
 
-    Array.forEach(attachmentContainers, function(attachmentContainer) {
-      var doc = attachmentContainer.contentDocument;
-      doc.documentElement.lang = navigator.mozL10n.language.code;
-      doc.documentElement.dir = navigator.mozL10n.language.direction;
-      navigator.mozL10n.translateFragment(doc.body);
+    return document.l10n.languages.then(([lang]) => {
```

Comment:
From the code of l20n, it looks like languages could be "not defined", because it's set in `changeLanguage` only. Wondering if you could set a promise in register that you'd resolve in changeLanguage